av/container/input.pyx

```diff
@@ -143,6 +136,19 @@ cdef class InputContainer(Container):
                 except EOFError:
                     break
 
+                if self.ptr.nb_streams != len(self.streams):
+                    self.streams = StreamContainer()
+                    for i in range(self.ptr.nb_streams):
+                        self.streams.add_stream(wrap_stream(self, self.ptr.streams[i]))
+
+                for i in range(self.ptr.nb_streams):
+                    include_stream[i] = False
```

Comment:
How do we know `include_stream` is the correct size, it only seems to be allocated once?

Reply:
I should re-allocate `include_stream`. Thanks.